test/compilable/crlf.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
```

Comment:
Done. Didn't add one initially because none of the existing `.sh` files had one.